### ISSUE TITLE
Saving a file with CTRL+S consistently crashes Atom (Windows 7 Enterprise Service Pack 1)

### ISSUE THREAD

**Issue by habi** [atom/atom#11911]
### Description

I downloaded a fresh installation of Atom and edited my first (Markdown) file with it.
Saving the file with CTRL+S consistently crashes Atom.
Not a very good first interaction with the editor on Windows...
### Steps to Reproduce
1. Download Atom
2. Install on Windows
3. Edit file
4. Press CTRL+S (In both 'normal' and 'safe' mode)

**Expected behavior:** The file is saved

**Actual behavior:** The editor crashes
### Versions

![clipboard01](https://cloud.githubusercontent.com/assets/1651235/15818611/ae874164-2bde-11e6-851e-c050a49694f4.jpg)
![clipboard02](https://cloud.githubusercontent.com/assets/1651235/15818610/ae86e8ae-2bde-11e6-99d7-9ae4128d4127.jpg)

**Comment by 50Wliu** [comment]
@habi Can you try the Atom beta (https://atom.io/beta) and see if the error persists?  Sorry for the bad first impression :frowning:.

**Comment by gabrielseco** [comment]
In my windows 10, I think Atom got updated and now it's crashing sometimes after saving.

My atom's  version is 1.7.4

**Comment by habi** [comment]
> @habi Can you try the Atom beta (https://atom.io/beta) and see if the error persists? Sorry for the bad first impression 😦.

I can, but not until Wednesday, when I’m physically back at work.

And don’t worry about the first impression, I’ve been using Atom for a good year on Linux and (due to a job change) am now “forced” to use it under Windows...

**Comment by igorkoudrik** [comment]
Hi, I have the same Windows machine setup as @habi and Atom crashes the same way whenever I try to save a document  (both by ctrl S and save from meny). I have tried both current version and beta version with the same result:(

**Comment by hakur0** [comment]
Getting the same issue. Gonna install the beta later and give some feedback on it. Also, note that this doesn't happen every time I save using CTRL + S.

My system and Atom version, in case it's useful info:
![versions](http://i.imgur.com/4ASQn3C.png)

**Comment by 50Wliu** [comment]
Can anyone who is experiencing the crashes please upload the latest `.dmp` file from `%TEMP%\Atom Crashes` here?  You'll have to zip it in order for GitHub to accept it.

**Comment by hakur0** [comment]
Strangely enough, there ain't any dump files since last month, and Atom crashed a few times since then. 

[Here's the latest DMP file](https://github.com/atom/atom/files/303472/9bdce308-ac4a-4ed1-bc5d-1e31e4d002b5.zip), in case the modified date is wrong, and [here's the operation_log.txt](https://github.com/atom/atom/files/303471/operation_log2.zip), which was updated today, so maybe it'll help?

**Comment by igorkoudrik** [comment]
Hi @50Wliu, here come atom crashes from my machine. Atom crashes always when I try to save a file on a network drive. It is works fine when saving on local disk. 
[Atom Crashes.zip](https://github.com/atom/atom/files/303652/Atom.Crashes.zip)

**Comment by habi** [comment]
Here's my latest crash file.
I'm also (trying) to save a file to a network drive.
Atom is also fine if I edit files on the local disc.

[0f3dc767-7673-4b3b-b1ae-0f9f9e8af4bd.zip](https://github.com/atom/atom/files/304210/0f3dc767-7673-4b3b-b1ae-0f9f9e8af4bd.zip)

**Comment by habi** [comment]
When saving document on network shares Atom 1.11.1 *still* crashes, making it essentially useless.

**Comment by habi** [comment]
Any update on this?
I'd still like to use Atom, but the crash still happens with version 1.13.0.

**Comment by LimYoungChul** [comment]
I'm having the same issue, with crashes on save.

Happens both on Windows 8.1 and Windows 10.

**Comment by azastrael** [comment]
I'm having the same issue 😢

**Comment by ozo2003** [comment]
not gonna make a new issue, run in this same behaviour on my machine. 
apm  1.16.1  
npm  3.10.5  
node 6.9.5 x64  
python 2.7.12  
git 2.7.4  
Atom    : 1.16.0-beta0  
Electron: 1.3.13  
Chrome  : 52.0.2743.82  
Node    : 6.5.0  
OS: XUbuntu 16.04 64bit

**Comment by powellnathanj** [comment]
Save and Save As both crash.  I just tried Atom today on Fedora 25, and whatever the latest download is.  Googled the issue and landed here.

**Comment by msembinelli** [comment]
Happening to me as well!
Atom 1.18 ia32
Windows 10 64

**Comment by bradennapier** [comment]
Having the same issue but only on one of my projects.  Working now to determine what is missing that is causing it.

**Comment by bradennapier** [comment]
Ok so this appears to occur when you are specifying plugins and/or a configuration that is invalid or requesting plugins that do not exist.

Atom should likely handle this situation rather than simply crash.

I fixed this by simply taking those out of my .eslintrc and installing the ones i didnt have that I needed.

**Comment by habi** [comment]
Could someone tackle this bug?
It's been more than a year I've submitted it and Atom still crashed when I load a file from a network drive.

The issue has actually gotten *worse* (with a fresh install of 1.19.2).
Now, when I open a (markdown) file from a network drive, Atom shows me the file for about 10 seconds, then crashes...

**Comment by nicksawe** [comment]
I have the same issue ! Atom version 1.19.5 ia32 on windows 10 64 bit.

**Comment by charles-passille-smartnets** [comment]
@bradennapier 
You're onto something there, it's definitely only started happening to me since installing Linter.

**Comment by maxbrunsfeld** [comment]
@habi Is this still happening to you on the latest Atom? I thought it might be fixed via https://github.com/atom/node-pathwatcher/pull/123.

**Comment by habi** [comment]
No, with a fresh install (1.22.1) I don't see a crash anymore, but as soon as I start to edit a file, I run into several issues of [#1422](https://github.com/atom/atom/issues/14227)

**Comment by habi** [comment]
So, in my opinion this issue can be closed, I don't know about the other reporters in the thread...

**Comment by pejman-saberin** [comment]
I am still getting these crashes on save, anyone in here who found out how to fix this?

Atom    : 1.23.0

**Comment by ZEUSOFCS** [comment]
I am still getting these crashes on save, as well

Atom : 1.23.0
OS: Mac OS

**Comment by Crysitna** [comment]
same problem 
any solution so far?

**Comment by maxbrunsfeld** [comment]
Mac users - can you share crash logs from Console.app?

**Comment by LeoFeriyDust** [comment]
Hey, I found the problem, i think. 
In my case i was a package called "Phyton-autopep8" i just disabled it and the problem was gone :D

**Comment by SimonAlfie** [comment]
I am having the same issue with Atom 1.25.1 on Win 7  Service pack 1.

Ctrl-S save will occasionally crash Atom. Atom will normally restart from the dialog show, but sometimes it repeatedly crashes on restart, requiring a full application restart.

These are the crash reports from the latest series of crashes I have experienced.

[b8bf3c28-ab70-4c81-969b-a54bcdd38b1e.txt](https://github.com/atom/atom/files/1938563/b8bf3c28-ab70-4c81-969b-a54bcdd38b1e.txt)

[737e3a46-b68d-4d24-8dce-2212f43dbe99.txt](https://github.com/atom/atom/files/1938564/737e3a46-b68d-4d24-8dce-2212f43dbe99.txt)

[eb022e8f-96e3-44a5-b241-4f94cfbfda2e.txt](https://github.com/atom/atom/files/1938565/eb022e8f-96e3-44a5-b241-4f94cfbfda2e.txt)

[2ace6cc2-1520-4ab3-a788-7920b9ed5842.txt](https://github.com/atom/atom/files/1938566/2ace6cc2-1520-4ab3-a788-7920b9ed5842.txt)

What is the easiest way to run Atom under a debug environment? I'm happy to build a version and run that while working to get more information on this crash. It is very annoying! Just point me in the direction of a page listing the required tools etc. and I will crack on.

Simon.

**Comment by Chistabo** [comment]
win 10 laptop - happens here also, when 'save all', and more than one file (opened in atom) has changes to save (at same time).
